### PR TITLE
Updating with two GeoPoints fails correctly

### PR DESCRIPTION
**Pull request by aontas**
Previously it would fail with an internal server error.

### CODE REVIEW

**Comment by aontas** [comment]
Alright, gotta run all the tests. Noted.

**Comment by aontas** [comment]
I have disabled the new test on postgres databases as I do not have the expertise to add something appropriate.

**Comment by aontas** [comment]
Updated to include badge count updating fixes.

File: spec/PushController.spec.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const successfulIOS = function(body, installations) {
   return Promise.all(promises);
 }
 
-describe('PushController', () => {
+fdescribe('PushController', () => {
```
Comment (markmssd) [inline spec/PushController.spec.js]: I think you forgot to remove this `f` :P

Comment (aontas) [reply spec/PushController.spec.js]: Seems to be my super power.

**Comment by flovilmart** [comment]
@aontas ??

**Comment by aontas** [comment]
This has been reverted to just having the GeoPoint changes. However, I cannot fetch parse-server as there is some sort of error with the remote.

ie,
```
git clone https://github.com/parse-community/parse-server.git test-parse-server
Cloning into 'test-parse-server'...
remote: Counting objects: 11508, done.
remote: Compressing objects: 100% (74/74), done.
remote: fatal: unable to read d8011799593281aae5698e46534eaa87ec862b3c
remote: aborting due to possible repository corruption on the remote side.
fatal: early EOF
fatal: index-pack failed
```
and note that this is a fresh clone..

So if you know how to fix this, that would be great, otherwise there doesn't seem much I can do until GitHub support gets back to me.

**Comment by dplewis** [comment]
@aontas This PR: https://github.com/parse-community/parse-server/pull/4069 allows for multiple `GeoPoints` so errors won't happen.

**Comment by flovilmart** [comment]
@aontas, i’ve See. It too, not sure what to do. Github can’t handle our scale :p

**Comment by aontas** [comment]
@flovilmart Let me know if you prefer #4069. And by that I mean you can close this ticket with a comment saying that that PR supersedes this one. Alternatively, you can accept and merge this, then have the other merged in on top.

**Comment by aontas** [comment]
@flovilmart Given you are adding #4069 to version 3, I'd suggest that it is probably a good idea to include this PR in one of the final 2.x releases.

**Comment by flovilmart** [comment]
Yep, that’s probably a good idea!

**Pull request merged**